DataDog/libddwaf-rb
# [APPSEC-55308] Add libddwaf releases checksums

**What does this PR do?**

This PR will add new Rake task `libddwaf:download_checksums` which will download last 100 releases checksums for specific platforms and store them into a single checksum file.

**Motivation**

Move checksums out of the Rakefile and automate checksum gathering

**How to test the change?**

1. Pull the branch
2. Create new token (see error message on 1st run)
3. run `bundle exec rake libddwaf:download_checksums`
4. run `bundle exec rake libddwaf:extract`
5. run `bundle exec rake binary`
